dashboard/app/views/pd/teachercon1819_registration/submitted.html.haml

```diff
@@ -1,7 +1,13 @@
 - content_for(:head) do
   = stylesheet_link_tag 'css/pd', media: 'all'
 
+- email = "teacher@code.org" if @application.application_type == "Teacher"
+- email = "facilitators@code.org" if @application.application_type == "Facilitator"
+- email = "partner@code.org" if @application.application_type == "Partner"
```

Comment:
Should this be determined in the controller, and passed here as a `@` parameter?